doc/man3/BIO_read.pod

```diff
@@ -25,8 +25,9 @@ BIO_read_ex() attempts to read I<dlen> bytes from BIO I<b> and places the data
 in I<data>. If any bytes were successfully read then the number of bytes read is
 stored in I<*readbytes>.
 
-BIO_write_ex() attempts to write I<dlen> bytes from I<data> to BIO I<b>. If
-successful then the number of bytes written is stored in I<*written>.
+BIO_write_ex() attempts to write I<dlen> bytes from I<data> to BIO I<b>.
+If successful then the number of bytes written is stored in I<*written>
+unless I<written> is NULL. This may be 0 in case I<dlen> is 0 or I<b> is NULL.
```

Comment:
I don't think that last sentence is necessary.

Reply:
Well, the `I<dlen> is 0` may be omitted in this context,
but the rest regarding `I<b> is NULL` is helpful to point out that `b` may be NULL.
In total, the sentence is analogous to a similar one in `BIO_write()`,
but if you prefer I can drop the `I<dlen> is 0 or` part.

Reply:
Please drop the dlen part. And yes, keep the `I<b> is NULL.` as that is non-obvious.

Reply:
All right, changed the sentence to: 
```
 No data is written if I<b> is NULL.
 ```